languages/rust/exercises/concept/entry-api/tests/entry_api.rs

```diff
@@ -0,0 +1,28 @@
+use entry_api::*;
+
+#[test]
+fn test_fn_returns_true_for_good_input() {
+    let magazine = &["give", "me", "one", "grand", "today"];
+    let note = &["give", "one", "grand", "today"];
```

Comment:
```suggestion
    let magazine = "The metro orchestra unveiled its new grand piano today. Its donor paraphrased Nathan Hale: \"I only regret that I have but one to give \"".split_whitespace();
    let note = &["give", "one", "grand", "today."];
```

Rewrite the magazine to be more plausible as a magazine article. Reorders the target words: students should not depend on their being in sequence.